specification/source/glossary.rst

```diff
@@ -70,3 +85,13 @@ the Open Building Controls project.
    Mode
      In CDL, by mode we mean a signal that can take on multiple distinct
      values, such as ``On``, ``Off``, ``PreCool``.
+
+   Non-HVAC System
+     Any non-HVAC plant coupled with the control system.
+
+   Plant
```

Comment:
@mwetter I see some ambiguity in your comment. Would you like us to use term ```plant``` and ```controller``` separately (which is how our documentation currently handles it, including this PR's addition to it), or would you prefer the term ```plant``` to include both the ```controller``` and the controlled plant, which is how ISO 16484-2 defines it (see below)?
```
3.149
plant
physical unit for a comprehensive process including the dedicated functional unit for control (3.51)

3.51
control
purposeful action on or in a process to meet specified objectives
```